satellite/marketingweb/server.go

```diff
@@ -165,6 +171,10 @@ func (s *Server) parseTemplates() (err error) {
 	return nil
 }
 
+func (s *Server) generatePartnerLink(offerName string) ([]string, error) {
+	return s.partners.GeneratePartnerLink(context.TODO(), offerName)
```

Comment:
Because the `generatePartnerLink` is called from inside a template and the threading context through template exec is more complicated than adding an argument.